source/edge-server/configure.txt

```diff
@@ -496,4 +414,143 @@ Upgrade Edge Server Procedure
+      button to get a new token, and copy it. You won't be able to see this
+      token again, and must generate a new token if you do not copy it.
+
+   .. step:: Supply the New Edge Server Token
+
+      Use ``edgectl`` to be prompted for a new token from the Edge Server 
+      instance:
+
+      .. code-block:: shell
+
+         edgectl --update-token=true
+
+      This triggers ``edgectl`` to prompt you for the token. When you provide 
+      it, the Edge Server instance exchanges this token for a secret. It uses 
+      the secret to authenticate with Atlas when it syncs data.
+
+      .. important:: Required network access
+
+         To exchange the token for an API key, ``edgectl`` must have network
+         access. You cannot cycle the API key if ``edgectl`` is not online.
+
+.. _specify-edge-server-query:
+
+Specify the Edge Server's Query
+-------------------------------
+
+You can use the ``--query`` argument with the ``edgectl configure`` command
+to specify the Edge Server instance Sync query.
+
+The ``--query`` for an Edge Server instance defaults to ``"*"``, which 
+automatically syncs all data in all of the collections in your 
+:ref:`schema <schemas>`.
+
+If you would like to filter the data that syncs to your Edge Server, you have
+two options:
+
+- Pass a simple string query. For example, ``--query="truepredicate"``.
+- Pass an object containing one or more collections and queries to sync a 
+  subset of your data with Edge Server. This may resemble:
+
+  .. code-block:: json
+
+     {
+        "YOUR-COLLECTION-NAME": "YOUR-QUERY",
+        "YOUR-COLLECTION-NAME": "YOUR-QUERY"
+     }
+
+  - ``YOUR-COLLECTION-NAME``: The name of a collection in your database. 
+    If you're using a template app, this is the ``Item`` collection. Your Edge 
+    Server query may contain many collections.
+
+  - ``YOUR-QUERY``: The query that determines which documents the Edge 
+    Server can sync with a collection. You can use :ref:`Realm Query Language <rql>` 
+    to define this query, with the same caveats around :ref:`Flexible Sync RQL 
+    Limitations <flexible-sync-rql-limitations>`.
+
+.. _edge-server-query-examples:
+
+Edge Server Query Examples
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Sync All Documents
+```````````````````
+
+You can sync all documents between Edge Server instance and Atlas. This uses 
+the ``truepredicate`` or ``"*"`` query. Because this is a simple query, you 
+can pass it as a string with the ``edgectl --query`` argument.
+
+.. code-block:: shell
+
+   edgectl --query="truepredicate"
+
+Sync a Subset of Documents
+``````````````````````````
+
+You might only want to sync a subset of documents with an Edge Server instance.
+For example, for an instance located in a specific store, you might want to
+sync only the documents relevant to that store. An RQL query for that might
+look similar to:
+
+.. code-block:: shell
+
+   edgectl --query="store_id == 42"
+
+Specify Different Queries for Multiple Collections
```

Comment:
this may be subject to change in the next couple of days. the current proposal is to either allow
```
--query-file query.json
```
or
```
--query "table:store_id==42" --query "table2:truepredicate"
```

Reply:
Ok, I've added a todo that I'll carry over to the feature branch to update this once the details are finalized. Thanks!